src/licensedcode/detection.py

```diff
@@ -155,7 +268,7 @@ def relevance(self):
         This is computed as the relevance of the sum of the underlying matches
         rule length.
         """
-        return compute_relevance(self.rules_len)
+        return compute_relevance(self.rules_length)
```

Comment:
Are you sure? `self.rules_length` is a  function 
```suggestion
        return compute_relevance(self.rules_length())
```

Reply:
Yes, thanks. Updating this.